src/utils/settings.py

```diff
@@ -13,23 +15,50 @@ def is_running_pytest() -> bool:
     return "pytest" in sys.modules
 
 
-project_root = os.path.dirname(os.path.abspath(__file__))
+def load_env_from_json() -> None:
+    """Load the configuration from the config.json file."""
+    default_config_path = Path(__file__).parent.parent.parent / "config" / "config.json"
+
+    config_path = Path(os.getenv("AICORE_HOME", default_config_path))
+
+    if os.path.exists(config_path):
+        try:
+            # Load the configuration from the given path and set the environment variables.
+            with open(config_path) as file:
+                config_file = json.load(file)
+
+                # Set environment variables for all keys except "models"
+                for key, value in config_file.items():
+                    if key != "models":  # Skip models
+                        os.environ[key] = str(value)
+        except json.JSONDecodeError as e:
+            logging.error(f"Invalid JSON format in config file {config_file}: {e}")
+            raise
+        except Exception as e:
+            logging.error(f"Error loading config from {config_path}: {e}")
+            raise
+    else:
+        logging.error(f"Config file not found at {config_path}.")
```

Comment:
I think we need to raise an exception here. If a config file cannot be found, the app cannot start anyway.

Reply:
done

Reply:
we should make it work everywhere. I can help you with that.